modules/340-monitoring-deckhouse/monitoring/prometheus-rules/alerting.yaml

```diff
@@ -117,3 +117,26 @@
         Please run `kubectl annotate mr {{ $labels.name }} modules.deckhouse.io/approved="true"` for confirmation.
       summary: |
         Module release is waiting for manual approval.
+  - alert: D8HasModuleConfigAllowedToDisable
+    expr: max by (module) (d8_moduleconfig_allowed_to_disable) >= 1
+    for: 3m
+    labels:
+      severity_level: "4"
+      d8_module: deckhouse
+      d8_component: deckhouse
+      tier: cluster
+    annotations:
+      plk_markup_format: "markdown"
+      plk_protocol_version: "1"
+      description: |
+        Module Configuration is waiting for disable.
+
+        It is recommended to keep clean your module configurations from approve annotations.
+
+        If you ignore this alert and didn't clean annotation - it may may result in a module being accidentally removed from the cluster.
```

Comment:
```suggestion
        If you ignore this warning and do not clear the annotation, it may cause the module to be accidentally removed from the cluster.
```

Reply:
done